src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/TypesThatOwnDisposableFieldsShouldBeDisposableTests.cs

```diff
@@ -44,25 +43,67 @@ public class NoDisposeClass
         [Fact]
         public async Task CA1001CSharpTestWithFieldInitAndNoDisposeMethodAsync()
         {
-            await VerifyCS.VerifyAnalyzerAsync(@"
+            await VerifyCS.VerifyCodeFixAsync(@"
 using System.IO;
 
-    public class NoDisposeClass
+    public class [|NoDisposeClass|]
     {
         FileStream newFile1, newFile2 = new FileStream(""data.txt"", FileMode.Append);
     }
+", @"
+using System.IO;
+
+    public class NoDisposeClass : System.IDisposable
```

Comment:
```suggestion
public class NoDisposeClass : System.IDisposable
```

Reply:
Up to you, if you want to fix these indents or not (a few more tests have the same issue).

Reply:
Yeah I saw they were copied hence my "optional" suggestion :)

Reply:
I wanted to do that, but the reason didn't is I usually see pushback against those kind of changes (at least in dotnet/roslyn).

I'd be definitely happy to fix formatting if @sharwell/@mavasani agrees :)